File: Java/IntegerListFromUsp/src/Exercicio02.java
import java.util.Scanner;

public class Exercicio02 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite um número: ");
        scanner.reset();
        int numero = scanner.nextInt();

        int soma = 0;
        for (int i = 1; i <= numero; i++) {
            soma += i;
        }
        System.out.print("A soma dos primeiros %d números inteiros positivos é: %d".formatted(numero, soma));
    }
}

File: CPP/IntegerListFromUsp/src/Exercicio02.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cout << "Digite um inteiro positivo n: ";
    cin >> n;

    long long soma = 0;
    for (int i = 1; i <= n; i++) {
        soma += i;
    }
    cout << "Soma dos " << n << " primeiros inteiros positivos: " << soma << endl;
    return 0;
}
